api/src/runtime.rs

```diff
@@ -27,16 +27,24 @@ pub fn bootstrap(
         .thread_name("api")
         .enable_all()
         .build()
-        .expect("[api] failed to create runtime");
+        .context("[api] failed to create runtime")?;
+    let context = Context::new(chain_id, db, mp_sender, config.clone());
 
-    let api = WebServer::from(config.api.clone());
-    let node_config = config.clone();
+    // Poem will run on a different port.
+    let poem_address = attach_poem_to_runtime(&runtime, context.clone(), config)
+        .context("Failed to attach poem to runtime")?;
```

Comment:
We should make sure we open the port on 127.0.0.1 then

Reply:
Yeah I'll need to experiment with the various deployments to make sure this works. If we need to open a port, I'll have to make this a predefined port, not a randomly selected one.